src/generators/node_graphql/utilities/GraphQLSchemaConfig.js

```diff
@@ -42,50 +48,53 @@ class GraphQLQuery {
       operation,
       resource,
       service,
-      isPrimaryGetter
+      isPrimaryGetter,
     };
   }
 
   get name() {
-    const { path, resultType, } = this.config.operation;
+    const { path, resultType } = this.config.operation;
     const staticParts = path.split('/').filter(x => x.length > 0 && x[0] !== ':');
     const queryParts = path.split('/').filter(x => x.length > 0 && x[0] === ':');
 
-    // get multiple instances of this resource
+    /* eslint-disable padded-blocks */
+
+    // Get multiple instances of this resource
     if (isEnclosingType(resultType)
      && getBaseType(resultType).fullyQualifiedType.fullyQualifiedType.match(strOrVersion(this.config.resource.type))) {
 
       return camelCase(this.config.resource.plural);
 
-    // get a single instance of this resource
+    // Get a single instance of this resource
     } else if (!isEnclosingType(resultType)
             && resultType.fullyQualifiedType.fullyQualifiedType.match(strOrVersion(this.config.resource.type))) {
 
       if (this.config.isPrimaryGetter) {
         return camelCase(this.config.resource.type.shortName);
-      } else {
+      } else { // eslint-disable-line no-else-return
```

Comment:
Should respect rules 😅

Reply:
I think it looks better if I don't follow this rule...